Source/IntegrationTests/TestFiles/LitTests/LitTest/ast/overrides.dfy

```diff
@@ -0,0 +1,43 @@
+// RUN: %testDafnyForEachCompiler %s
+ 
+trait T {
+  method foo() returns (r: int)  {
+    return 3;
+  }
+}
+
+trait T2 extends T {
+  method foo() returns (r: int)  {
```

Comment:
Rename `foo` `Foo` here and throughout to adhere to standard Dafny naming conventions.